### ISSUE TITLE
I can't upload any mods

### ISSUE THREAD

**Issue by Uns0lved** [bluemagic123/tModLoader#3873]
### Version

Latest 1.4.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Mod capability as a Modder

### Description

when I try upload a mod to the workshop I get a empty job that only I can see and error in TML "K_EResultTimeout". VPN doen't fix that. Steam is not blocked on my current network.
![dotnet_zAGneJ1vlj](https://github.com/tModLoader/tModLoader/assets/55342617/d0db98c3-397b-482f-b5ea-2ed264ad8963)
![steamwebhelper_f6tdXuN7lE](https://github.com/tModLoader/tModLoader/assets/55342617/df4b5334-32cb-4ea1-a73b-cc241f8f8b7a)

### Log File

[client.log](https://github.com/tModLoader/tModLoader/files/12855011/client.log)
[environment-client.log](https://github.com/tModLoader/tModLoader/files/12855012/environment-client.log)
[Launch.log](https://github.com/tModLoader/tModLoader/files/12855013/Launch.log)
[Natives.log](https://github.com/tModLoader/tModLoader/files/12855014/Natives.log)
[workshop_log.txt](https://github.com/tModLoader/tModLoader/files/12855031/workshop_log.txt)

### Steps to reproduce

Build + Reload any mod
Try to publish it

### Expected Behavior

The mod should be published in the workshop

### Actual Behavior

I get error "K_EResultTimeout"

### Reproduction frequency

All the time

**Comment by JavidPack** [comment]
Did you ever figure this out? I see that you ran into this issue on a Tuesday, I think the Steam workshop sometimes has maintenance on Tuesdays. If it works now, maybe we can find a way to better report this error in this situation.

**Comment by Uns0lved** [comment]
I tried to do it not only on Tuesday. I still can't publish any mod and get the same error

**Comment by JavidPack** [comment]
The December release has some workshop fixes that might have fixed this for you. Give it a try. If that still doesn't work, we might need to see C:/Program Files (x86)/Steam/logs/workshop_log.txt and new logs.

**Comment by JavidPack** [comment]
This issue has either no recent activity or no reproducible steps, so it is being closed. We assume this issue has solved itself in some manner. Feel free to open a new issue if the issue persists and fill out the issue template completely.
